app/lib/backend_api_logic/service_extension.rb

```diff
@@ -17,6 +19,8 @@ module ServiceExtension
       end, class_name: 'Metric'
     end
 
+    # This is used by db/migrate/20190716110520_create_the_backend_apis_of_services.rb
+    # We should Remove this after 2.7 is released
```

Comment:
Good, but put a `TODO``in the comment, and better if u create a jira issue, because this looks like the kind of thing that stays there for ages...